pkg/codegen/python/doc.go

```diff
@@ -47,16 +47,20 @@ func (d DocLanguageHelper) GetDocLinkForResourceType(pkg *schema.Package, modNam
 
 	var path string
 	var fqdnTypeName string
+	namespace := "pulumi"
+	if pkg.Namespace != "" {
+		namespace = strings.ReplaceAll(pkg.Namespace, "_", "-")
```

Comment:
Should we be calling `pyPack` here as well?

Reply:
Good idea!  Will change.